app/src/main/java/com/forrestguice/suntimeswidget/HelpDialog.java

```diff
@@ -96,20 +96,20 @@ public void onResume()
         expandSheet(getDialog());
     }
 
-    private void expandSheet(DialogInterface dialog)
-    {
```

Comment:
Strive to make your changes as succinct as possible. The only meaningful change here is the addition of the `static` keyword. Avoid whitespace only changes (style) as this just makes the diff harder to read.